assets/js/data/cart/resolvers.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies
+ */
+import { select, apiFetch } from '@wordpress/data-controls';
+import { camelCase, mapKeys } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { receiveCart, receiveError } from './actions';
+import { STORE_KEY, MISSING_ROUTE_ERROR, CART_API_ERROR } from './constants';
+import { STORE_KEY as SCHEMA_STORE_KEY } from '../schema/constants';
+
+/**
+ * Resolver for retrieving all cart data.
+ */
+export function* getCartData() {
+	const route = yield select(
+		SCHEMA_STORE_KEY,
+		'getRoute',
+		'/wc/store',
+		'cart'
+	);
+
+	if ( ! route ) {
+		yield receiveError( MISSING_ROUTE_ERROR );
+		return;
+	}
```

Comment:
Being this is a very specific resolver, I would just use the route directly in the call to `apiFetch` and bypass getting the route from the schema store.